arcaflow_plugin_pcp/pcp_schema.py

```diff
@@ -13,8 +13,7 @@ class PcpInputParams:
             " space-separated string."
         ),
     ] = (
-        "kernel.cpu.util kernel.all.load"
-        " mem.util.used disk.all.read disk.all.write"
+        "kernel.cpu.util kernel.all.load" " mem.util.used disk.all.read disk.all.write"
```

Comment:
You should probably remove the embedded `" "` since the string is now all on one line.